frontend/ui/impl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/table/EnhancedGraphTable.java

```diff
@@ -147,12 +152,30 @@ public EnhancedGraphTable(Project project) {
 
     subscribeToGitRepositoryFilesChanges();
     subscribeToSelectedGitRepositoryChange();
+    subscribeToMacheteFileChange();
   }
 
   private IGitRepositorySelectionProvider getGitRepositorySelectionProvider() {
     return project.getService(SelectedGitRepositoryProvider.class).getGitRepositorySelectionProvider();
   }
 
+  private void subscribeToMacheteFileChange() {
```

Comment:
I do not see a reason to keep it within GitMachetePanel. actually here it seems to be more reasonable. and it will be more convenient for future changes in scope of unmanaged branch notification expiration